docs-src/modules/developing/pages/materializations.adoc

```diff
@@ -157,5 +157,10 @@ session = LocalBuilder().build()
 materialization.delete_materialization("PurchaseStats")
 ----
 
+[source,bash]
+----
```

Comment:
I think we should call out that this is a separate way of deleting a materialization. I think as a first pass (until we have tabs) we could add a title to the example, like

```
[source,Python]
.Deleting a materialization with Python
----
from kaskada import materialization
from kaskada.api.session import LocalBuilder

session = LocalBuilder().build()

materialization.delete_materialization("PurchaseStats")
----

[source,bash]
.Deleting a materialization with the CLI
----
./kaskada-cli materialization delete -m PurchaseStats
----
```

As is, it sort of reads like you need to _both_ delete it in Python and then run a CLI command.